docs/conf.py

```diff
@@ -0,0 +1,33 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+project = 'QE Metrics'
+copyright = '2023, ilausuch'
```

Comment:
Let's set a opensource copyright: Apache, Mit, not sure

Reply:
this is a different variable. i can added

Reply:
Actually this is not taking the Licence. So i used `SUSE LLT` as we do for os-autoinst